GitCommands/Git/Commands.Arguments.cs

```diff
@@ -516,43 +516,47 @@ public static ArgumentString PushTag(string path, string tag, bool all, ForcePus
             };
         }
 
-        public static ArgumentString Rebase(
-            string? branch, bool interactive, bool preserveMerges, bool autosquash, bool autoStash, bool ignoreDate, bool committerDateIsAuthorDate, string? from = null, string? onto = null, bool supportRebaseMerges = true)
+        public static ArgumentString Rebase(RebaseOptions rebaseOptions)
```

Comment:
```suggestion
        public static ArgumentString Rebase(in RebaseOptions rebaseOptions)
```
in order to emphasize that the options are read-only (some of the properties are declared with setter).